src/pages/AddGamePage.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components/macro'
+import Search from '../components/Search'
+
+//const api_key = process.env.REACT_APP_API_KEY;
+
+export default function AddGamePage({ onSubmit, games }) {
+  return (
+    <Container>
+      <Search onSubmit={onSubmit} />
+      <List>
+        {games.map(game => (
+          <Li backgroundImage={game.background_image}>{game.name}</Li>
```

Comment:
Please name it: `ListItem` this name is not good
```suggestion
          <ListItem backgroundImage={game.background_image}>{game.name}</ListItem>
```